environment.yml

```diff
@@ -63,10 +63,10 @@ dependencies:
   - xgboost>1.6.1  # github.com/ESMValGroup/ESMValTool/issues/2779
   - xlsxwriter
   # Python packages needed for testing
+  - flake8 <5
```

Comment:
Was there any other reason to pin flake8 than that it is not compatible with pytest-flake8?

Reply:
that, my sir, is a **very** good question - I think people reported issues with 5 or morem but reckon all of them were related to the plugin, let's unpin later, am still traumatized by any sort of flake8 issue :laughing: